tests/tasks/rss/__init__.py

```diff
@@ -0,0 +1,3 @@
+import pytest
+
+pytestmark = pytest.importorskip("feedparser")
```

Comment:
```suggestion
pytest.importorskip("feedparser")
```
to prevent more work in #814